src/main/kotlin/Board.kt

```diff
@@ -0,0 +1,108 @@
+class Board(initialPositions: Map<Char, List<String>> = mapOf()) {
+    private var turn: Int = 0
+    private val boardSpots = arrayOf(charArrayOf(' ', ' ', ' '), charArrayOf(' ', ' ', ' '), charArrayOf(' ', ' ', ' '))
+
+    init {
+        initialPositions.forEach { (xOr0, placements) ->
+            placements.forEach { move ->
+                val (row, col) = parseMove(move)
+                boardSpots[row][col] = xOr0
+            }
+        }
+    }
```

Comment:
This allowed me to set an initial state of  the board so I could test without having to setup a full gameplay and add turns that weren't relevant just to set the data up.